File: core/src/main/java/com/ustadmobile/core/view/PersonPictureDialogView.java
package com.ustadmobile.core.view;

public interface PersonPictureDialogView extends UstadView, DismissableDialog{
    String VIEW_NAME="PersonPictureDialogView";
    String ARG_PERSON_IMAGE_PATH = "PersonImagePath";
    String ARG_PERSON_UID = "PersonUid";
    void finish();
    void setPictureOnView(String imagePath);
    void showUpdateImageButton(boolean show);

}

File: core/src/commonMain/kotlin/com/ustadmobile/core/view/PersonPictureDialogView.kt
package com.ustadmobile.core.view

interface PersonPictureDialogView : UstadView, DismissableDialog {
    fun finish()
    fun setPictureOnView(imagePath: String)
    fun showUpdateImageButton(show: Boolean)

    companion object {
        val VIEW_NAME = "PersonPictureDialogView"
        val ARG_PERSON_IMAGE_PATH = "PersonImagePath"
        val ARG_PERSON_UID = "PersonUid"
    }

}
